docs/modules/csvimport.md

```diff
@@ -1,90 +1,93 @@
 # CSV Import
 
-The CSV Import module allows you to import items or users into your Omeka S install from a csv (comma separated values), tsv (tab-separated values) or odf (open document format) file. 
+The [CSV Import module](https://omeka.org/s/modules/CSVImport){target=_blank} allows you to import items, item sets, media, and users into your Omeka S install from a CSV (comma-separated values), TSV (tab-separated values), or ODF (open document format) file.
 
-Please note that an import must have a specific import type. 
+<!--- too much repetition in this page. need a "common settings" section first, then just the resource-type-specific comments in sections after that. --->
```

Comment:
Yeah, probably so. The redundancy is based on the fact that most people will only read about the kind of import they want to do. It could certainly be re-engineered to help streamline things.